backend/contentBackend.js

```diff
@@ -118,15 +127,15 @@ export async function updateContent(jwt, content) {
   const route = cleanRoute(parsed.attributes.route);
 
   try {
-    await update(`${contentPrefix}${route}`, (prevContent) => ({
+    await update(`${contentPrefix}${route}`, prevContent => ({
       ...parsed,
-      disabled: (prevContent ? prevContent.disabled : true),
+      disabled: prevContent ? prevContent.disabled : true,
     }));
   } catch (err) {
     console.error(err);
     // "err" object is currently wrapped by backend,
     // this makes it very hard to use
-    return potentialError || {
```

Comment:
Why did you  remove this?

Reply:
potentialError is undefined so was redundant.